spec/controllers/users_controller_spec.rb

```diff
@@ -10,6 +10,33 @@
     session[:user_id] = user.id
   end
 
+  describe '#create' do
+    let(:user_params) do
+      {
+        user: {
+          name: Faker::Name.name,
+          email: Faker::Internet.email,
+          password: 'Password123+',
+          password_confirmation: 'Password123+',
+          language: 'language'
+        }
+      }
+    end
+
+    it 'creates a current user if the new user is created' do
+      session[:user_id] = nil
+      create(:role, name: 'User') # Needed for admin#create
+      post :create, params: user_params
+      expect(session[:user_id]).to eql(User.last.id)
+    end
+
+    it 'creates a new user without changing the current user if the new user is created from a logged in user' do
```

Comment:
The tests description is setting that we're creating of a new user but we're not asserting that,